utils/queryFormatUtils.ts

```diff
@@ -53,15 +53,12 @@ export const convertValidateConsensusGenomeQuery = (query: string): string => {
   return (
     query
       // Replace Fed variables.
-      .replace(
-        /query [\s\S]*?{/,
-        "query ($where: WorkflowRunWhereClause) {",
-      )
+      .replace(/query [\s\S]*?{/, "query ($where: WorkflowRunWhereClause) {")
```

Comment:
same comment about regex, could we do it another way to add a comment explaining regex?